### ISSUE TITLE
[Flow] Show new updates on the feed without refresh

### ISSUE THREAD

**Issue by JorisBassle** [threefoldtech/twin_aydo#291]
use web sockets for this

**Comment by JorisBassle** [comment]
Need to wait on refactor for further development

**Comment by ken-de-moor** [comment]
Closing this feature as it's really too low prio and a nice-to-have.
